### PR TITLE
doc: Add doc for direct-load module.

### LINKED ISSUE #96: [subtask]: Add doc for direct-load module.
### Describe the subtask

Add doc for direct-load module.
### Parent issue

#92

### PR DESCRIPTION
**Pull request by yuanoOo**
## Summary

closes #96

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,10 +17,11 @@ Prerequisites
...
 This repository contains connectors as following:
 
-|          Connector          | OceanBase Compatible Mode |               Supported Features                |
-|-----------------------------|---------------------------|-------------------------------------------------|
-| Flink Connector: OceanBase  | MySQL, Oracle             | [Sink](docs/sink/flink-connector-oceanbase.md)  |
-| Flink Connector: OBKV HBase | OBKV HBase                | [Sink](docs/sink/flink-connector-obkv-hbase.md) |
+|               Connector                | OceanBase Compatible Mode |                    Supported Features                     |
+|----------------------------------------|---------------------------|-----------------------------------------------------------|
+| Flink Connector: OceanBase             | MySQL, Oracle             | [Sink](docs/sink/flink-connector-oceanbase.md)            |
+| Flink Connector: OceanBase Direct Load | MySQL                     | [Sink](docs/sink/flink-connector-oceanbase-directload.md) |
```
Comment (whhe) [inline README.md:23]: It is not accurate to say that it works for mysql mode, obkv-hase and obkv-redis mode of new 4.x versions can also use direct load. You can remove the compatible mode column and describe the features details in the 'Supported Features' column.

File: docs/sink/flink-connector-oceanbase-directload.md [hunk 2]
```diff
@@ -0,0 +1,427 @@
...
+                <td>Integer</td>
+                <td>Rpc port number used in direct-load.</td>
+            </tr>
+            <tr>
+                <td>username</td>
+                <td>Yes</td>
+                <td>Yes</td>
+                <td style="word-wrap: break-word;"></td>
+                <td>String</td>
+                <td>The username.</td>
```
Comment (whhe) [inline docs/sink/flink-connector-oceanbase-directload.md:299]: It's better to note users what kind of username is required. For now we have two versions:
1. Pure user name, the name of the database user, like 'root'.
2. Connection user name, which can be used to create the connection by client, like 'root@test' when directly connected to observer or 'root@test#obcluster' when connected to obproxy.
3. Full user name, like the username used to connect to obproxy, in format 'root@test#obcluster' or 'obcluster:test:root'.

Comment (yuanoOo) [reply docs/sink/flink-connector-oceanbase-directload.md]: Change to: The name of the database user, like 'root'. NOTE: Not the connection username like 'root@sys'.

File: README.md (hunk 1 above)
Comment (yuanoOo) [reply README.md]: Fixed.

**Review by whhe — APPROVED** [review]

**Pull request merged**